using_images/xpaas_images/sso.adoc

```diff
@@ -0,0 +1,166 @@
+
+== Prerequisites for Deploying the Red Hat Single Sign-On xPaaS Image
+The following is a list of prerequisites for using the Red Hat Single Sign-On xPaaS image:
+
+. *Acquire Red Hat Subscriptions* - Ensure that you have the relevant subscriptions for OpenShift as well as a subscription for xPaaS Middleware.
+. *Install OpenShift* - Before using the OpenShift xPaaS images, you must have an OpenShift environment installed and configured:
+.. The https://docs.openshift.com/enterprise/3.1/install_config/install/quick_install.html[Quick Installation] method allows you to install OpenShift using an interactive CLI utility.
+.. The https://docs.openshift.com/enterprise/3.1/install_config/install/advanced_install.html[Advanced Installation] method allows you to install OpenShift using a reference configuration. This method is best suited for production environments.
+. Ensure the *DNS* has been configured. This is required for communication between JBoss EAP and SSO, and for the requisite redirection. See https://docs.openshift.com/enterprise/3.1/install_config/install/prerequisites.html#prereq-dns[DNS] for more information.
+. *Install and Deploy Docker Registry* - Install the Docker Registry and then ensure that the Docker Registry is deployed to locally manage images:
++
+----
+$ oadm registry --config=/etc/origin/master/admin.kubeconfig \
+--credentials=/etc/origin/master/openshift-registry.kubeconfig
+----
++
+For more information, see https://docs.openshift.com/enterprise/3.1/install_config/install/docker_registry.html[Deploying a Docker Registry]
+. *Deploy a Router* - For more information, see https://docs.openshift.com/enterprise/3.1/install_config/install/deploy_router.html[Deploying a Router].
+. Ensure that you can run the _oc create_ command with https://docs.openshift.com/enterprise/3.1/architecture/additional_concepts/authorization.html#roles[cluster-admin] privileges.
+
+== Using the Red Hat Single Sign-On Image Streams and Application Templates
+The Red Hat xPaaS middleware images were
+link:../../install_config/imagestreams_templates.html[automatically created during the installation]
+of OpenShift along with the other default image streams and templates.
+
+== Preparing and Deploying the Red Hat Single Sign-On xPaaS Application Templates
+=== Using the OpenShift CLI
+Log in with a user that holds _cluster-admin_ privileges and perform the following in the {project-name} project space:
+
+. Prepare the JBoss EAP and SSO application service accounts and their associated secrets.
++
+----
+oc create -n {project-name} -f {application-templates_file_path}/secrets/eap-app-secret.json
+----
++
+----
+oc create -n {project-name} -f {application-templates_file_path}secrets/sso-app-secret.json
+----
+. Deploy one of the SSO application templates. This example deploys the *_sso70-postgresql_* template, which deploys two SSO pods, one SSO web server and one encrypted SSO web server, and a third pod for the PostgreSQL database.
```

Comment:
The commas sort of make it sound like there are perhaps seven pods. maybe put the middle aside into brackets.

"...which deploys two SSO pods (one SSO web server and one encrypted SSO web server) and a third pod for the PostgreSQL database."